### PR TITLE
Updated extent and added layers for Bavaria

### PR DESCRIPTION
**Pull request by ronitjadhav**
Updated the extent and added layers for Bavaria through QGIS project

### CODE REVIEW

File: geoportal/vars.yaml [hunk 1]
```diff
@@ -13,7 +13,8 @@ vars:
 
   resolutions: &resolutions [250, 100, 50, 20, 10, 5, 2, 1, 0.5, 0.25, 0.1, 0.05]
 
-  extent: &extent [2485071.54, 175346.36, 2828515.78, 1299941.84]
+  # extent: &extent [2485071.54, 175346.36, 2828515.78, 1299941.84]
+  extent: &extent [2485071.54, 175346.36, 3171913, 1709347]
```
Comment (sbrunner) [inline geoportal/vars.yaml:19]: Why change the extent?
What's the new extent?
Just a question, for me, it looks to be OK :-)

Comment (ronitjadhav) [reply geoportal/vars.yaml:19]: I added a few layers for Bavaria Germany, which required me to change the extent to make them visible on the map. I hope this clarifies the reason for the change.

Comment (sbrunner) [reply geoportal/vars.yaml:19]: OK, looks good to me :-)

**Review by sbrunner — APPROVED** [review]

**Comment by arnaud-morvan** [comment]
I think you openned your pull request on prod-2-8-advance instead of prod-2-8.

I suppose we should never open pull requests on prod-2-8-advance with changes in files existing in prod-2-8.
Instead you should open a pull request on prod-2-8 and those changed will be copied to prod-2-8-advance by GitHub Action.

You should rebase your last commit on top of prod-2-8, example:
```
git fetch origin
git rebase --onto prod-2-8 prod-2-8-advance add_layers_bavaria
git push -f origin add_layers_bavaria
```

This in this Pull Request on the right of Title, click edit and change base branch to prod-2-8.

Then I'm pretty sure the prod-2-8-advance branch with get updated by GitHub Action after merge.

Regards

**Comment by arnaud-morvan** [comment]
@sbrunner : Do you see something about those errors with snyk ?

It seems I cannot access the camptocamp org in snyk.

**Comment by ronitjadhav** [comment]
@sbrunner Do you have any suggestions on how to resolve the issue?

**Comment by sbrunner** [comment]
Snyk says `Merge conflict: could not complete test` but there isn't any conflict => not pertinent => ignore it :-)

**Pull request merged**